crates/orchestrator/src/lib.rs

```diff
@@ -173,7 +152,6 @@ impl<KEY: SignatureKey + 'static> OrchestratorState<KEY> {
             transactions_per_round: self.config.transactions_per_round,
             transaction_size: self.bench_results.transaction_size_in_bytes,
             rounds: self.config.rounds,
-            leader_election_type: OrchestratorState::<KEY>::election_type(),
```

Comment:
It would be better to leave this as it will show which `election_type` we use for benchmarking, but not an urgent thing if you'll add that back in with some small config changes.

Reply:
added back in in a slightly different form (taking the information from the nodes directly, rather than at compile-time in the orchestrator)